packages/webpack/src/runtime/runtimeBuilder.js

```diff
@@ -1,11 +1,55 @@
 const { RUNTIME_KEY } = require('../ENUM.json')
 const diag = require('../buildtime/diagnostics.js')
-const { assembleRuntime } = require('./assemble.js')
+const { assembleRuntime, prepareSource } = require('./assemble.js')
 const path = require('node:path')
 
-/** @import {LavaMoatPluginOptions} from '../buildtime/types' */
+const { RuntimeModule } = require('webpack')
+
+class VirtualRuntimeModule extends RuntimeModule {
+  /**
+   * @param {Object} options - The options for the VirtualRuntimeModule.
```

Comment:
(style suggestion)

The extra `-` is not treated as special by VSCode's on-hover documentation panel, which means at least in _that_ context it is unhelpful.  I would probably omit these.